src/drivers/spi/spi_flash.c

```diff
@@ -72,7 +72,7 @@ int spi_flash_cmd(const struct spi_slave *spi, u8 cmd, void *response, size_t le
 	return ret;
 }
 
-static int spi_flash_cmd_read(const struct spi_slave *spi, const u8 *cmd,
+int spi_flash_cmd_read(const struct spi_slave *spi, const u8 *cmd,
```

Comment:
@mek-x fix ident:

https://gist.github.com/davidzchen/9187984#file-sample-linux-c-L83

Reply:
it's consistent with other functions in this file